pyxem/signals/correlation2d.py

```diff
@@ -70,10 +70,10 @@ def get_angular_power(self, inplace=False, **kwargs):
         power = self.map(corr_to_power, inplace=inplace, **kwargs)
         if inplace:
             self.set_signal_type("power")
-            fourier_axis = self.axes_manager.signal_axes[1]
```

Comment:
Is this a bugfix?

Reply:
Yea, let me make a different pull request for that..